test/DatabaseTester.cs

```diff
@@ -121,10 +121,31 @@ public class DatabaseTester {
 			var result = db.ScriptCreate();
 			TestHelper.DropDb("TEST");
 
-			Assert.That(result, Is.StringContaining("CREATE  NONCLUSTERED INDEX [MyIndex] ON [dbo].[MyTable] ([Id]) WHERE ([EndDate] IS NULL)"));
+			Assert.That(result, Is.StringContaining("CREATE  NONCLUSTERED INDEX [MyIndex] ON [dbo].[MyTable] ([Id] ASC) WHERE ([EndDate] IS NULL)"));
 		}
 
-		[Test]
+        [Test]
+        public void TestViewIndexes()
+        {
+            TestHelper.DropDb("TEST");
+            TestHelper.ExecSql("CREATE DATABASE TEST", "");
+
+            TestHelper.ExecSql(@"CREATE TABLE MyTable (Id int, Name nvarchar(250), EndDate datetime)", "TEST");
+            TestHelper.ExecSql(@"CREATE VIEW dbo.MyView WITH SCHEMABINDING as SELECT t.Id, t.Name, t.EndDate from dbo.MyTable t", "TEST");
+            TestHelper.ExecSql(@"CREATE UNIQUE CLUSTERED INDEX MyIndex ON MyView (Id, Name)", "TEST");
+
+            var db = new Database("TEST")
+            {
+                Connection = TestHelper.GetConnString("TEST")
+            };
+            db.Load();
+            var result = db.ScriptCreate();
+            TestHelper.DropDb("TEST");
+
+            Assert.That(result, Is.StringContaining("CREATE UNIQUE CLUSTERED INDEX [MyIndex] ON [dbo].[MyView] ([Id] ASC, [Name] ASC)"));
```

Comment:
nitpick - you can use more direct methods for Asserting on strings  http://www.nunit.org/index.php?p=stringAssert&r=2.2.10

`StringAssert.Contains( string expected, string actual );`   and you will be get better generic error messages